scripts/build_project.sh

```diff
@@ -0,0 +1,43 @@
+# Author: Teghveer Singh Ateliey
+# Date: 2024-10-30
+
+#!/bin/bash
```

Comment:
check this, but I think this has to be the first line of the file to do anything